### ISSUE TITLE
[Task] Support pre-trained embeddings for ranking and session based models via the new dataloader functionality

### ISSUE THREAD

**Issue by EvenOldridge** [NVIDIA-Merlin/models#1043]
- [ ] Update the input block to support the pre-trained embeddings:
- [ ] Add a module to extract pre-trained embeddings

- [ ] Show an example of how to transform pre-trained embeddings before aggregating them with the other trainable embeddings in the model. Such as `Layer Normalization` or `MLP layer`. 
- [ ] Update DLRM, DeepFM, etc to include an optional projection layer for combining pretrained embeddings with the other embeddings in the model.

- [ ] Implement and evaluate different aggregation methods to combine pre-trained embeddings with the other trainable embeddings.

- [ ] Add an example demonstrating support for pretrained embeddings for DLRM or another non session based model.  Artificial example? KDD cup?

### Constraints
- Dataloader should support fixed-size 3D tensors. 
- Dataloader needs to add a tag to the pre-trained embeddings so that the T4R input module can differentiate between trainable features and pre-trained embeddings.

**Comment by gabrielspmoreira** [comment]
Closing this macro-task, as the sub-tasks have been listed in this [RMP](https://github.com/NVIDIA-Merlin/Merlin/issues/211)
